From c0e8110e6c75758c4567f8e713f26e5dbd88cc7c Mon Sep 17 00:00:00 2001
From: Namjae Jeon <namjae.jeon@samsung.com>
Date: Wed, 17 Mar 2021 16:52:17 +0900
Subject: [PATCH] cifsd: fix WARNING: Title overline too short
Git-commit: c0e8110e6c75758c4567f8e713f26e5dbd88cc7c
Patch-mainline: v5.15-rc1
References: bsn#12
Modified-by-SEL: No


Stephen reported a warning message from cifsd.rst file.

 Documentation/filesystems/cifs/cifsd.rst:3: WARNING: Title overline too
 short.

Reported-by: Stephen Rothwell <sfr@canb.auug.org.au>
Signed-off-by: Namjae Jeon <namjae.jeon@samsung.com>
Signed-off-by: Steve French <stfrench@microsoft.com>
Signed-off-by: Guoqing Jiang <guoqing.jiang@suse.com>
---
 Documentation/filesystems/cifs/cifsd.rst | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/Documentation/filesystems/cifs/cifsd.rst b/Documentation/filesystems/cifs/cifsd.rst
index e0c33d03f290..af3589da6923 100644
--- a/Documentation/filesystems/cifs/cifsd.rst
+++ b/Documentation/filesystems/cifs/cifsd.rst
@@ -1,8 +1,8 @@
 .. SPDX-License-Identifier: GPL-2.0
 
-=========================
+==========================
 CIFSD - SMB3 Kernel Server
-=========================
+==========================
 
 CIFSD is a linux kernel server which implements SMB3 protocol in kernel space
 for sharing files over network.
-- 
2.33.0

